test/utils.test.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line spaced-comment
+/* eslint-env node, jest */
+/* global BigInt */
+
+"use strict";
+
+const utils = require("../lib/output/utils");
+
+describe("utils.js", () => {
+  describe("isObject", () => {
+    const BIG_INT_PLACEHOLDER = "BigInt placeholder";
+
+    /** @type {Array<string | number | bigint | boolean | symbol | null | undefined>} */
+    const primitives = [
+      123,
+      "string",
+      Symbol.iterator,
+      true,
+      BIG_INT_PLACEHOLDER,
+      null,
+      undefined
+    ];
+
+    // eslint-disable-next-line new-cap, valid-typeof
+    if (typeof BigInt === "function" && typeof BigInt("123") === "bigint") {
+      primitives.splice(
```

Comment:
Instead of creating adding a placeholder and replacing the placeholder if `BigInt` is defined, what do you think about just pushing a `BigInt` at the end of the `primitives` array if it is defined? I think it would greatly simplify the test code.

```js
const primitives = [ /*...*/ ];

if (typeof BigInt === "function") { 
  primitives.push(BigInt(123));
}
```

Reply:
Well, I&nbsp;want the&nbsp;test&nbsp;logs to&nbsp;show&nbsp;that the&nbsp;`BigInt`&nbsp;test was&nbsp;skipped.

Reply:
In that case, let's just do it the old fashioned way. Remove the `BIG_INT_PLACEHOLDER` trickery and instead just do

```js
const primitives = […];
for (const primitive of primitives) { … }

if (typeof BigInt === "function") { 
  test("bigint", () => {
    expect(utils.isObject(BigInt(123))).toBe(false);
  });
} else {
  test.skip("bigint");
}
```

Reply:
Actually, `test.skip`&nbsp;also&nbsp;takes the&nbsp;test&nbsp;case function&nbsp;argument.